server/selinux/candlepin.te

```diff
@@ -103,3 +115,7 @@ allow tomcat_t candlepin_var_lib_t:file { manage_file_perms };
 allow tomcat_t candlepin_var_log_t:dir { manage_dir_perms };
 allow tomcat_t candlepin_var_log_t:file { manage_file_perms };
 allow tomcat_t postgresql_port_t:tcp_socket name_connect;
+
+tunable_policy(`candlepin_can_bind_activemq_port',`
+	allow tomcat_t candlepin_activemq_port_t:tcp_socket name_bind;
```

Comment:
Indentation again. Looks like a tab instead of spaces?

Reply:
fixed